src/testing/localstorage.test.js

```diff
@@ -0,0 +1,45 @@
+import { save, load } from "../js/storage/index.js";
```

Comment:
I recommend you update the name of the file to logout.test.js because that is the main functionality you are testing here